src/smspark/constants.py

```diff
@@ -0,0 +1,5 @@
+
+DRIVER_MEM_INSTANCE_MEM_RATIO = 0.9
+DRIVER_MEM_OVERHEAD_PCT = 0.1
```

Comment:
This is not percent, it's also a ratio, isn't it? can we call it as such? same thing below.

Reply:
sure